plugins/Feedback/tests/Integration/ReferBannerTest.php

```diff
@@ -46,7 +45,11 @@ public function setUp(): void
     public function tearDown(): void
     {
         Option::deleteLike('Feedback.nextReferReminder.%');
-        $this->userModel->deleteUserOnly('user1');
+        try {
+            $this->userModel->deleteUserOnly('user1');
+        } catch (\Exception $e) {
+            // ignore possible errors triggered when the delete user event is posted
+        }
```

Comment:
One test didn't work on travis. The `deleteUserOnly` method fire a event, which fails somewhere when no user is logged in. As it's not relevant in this case, it can be simply ignored.